cpp/tools/clustertool_main.cc

```diff
@@ -47,64 +66,121 @@ void Usage() {
             << "  clustertool [flags] <command> [command opts]\n"
             << "\n"
             << "Commands:\n"
-            << "  initlog     Initialise a new log.\n";
+            << "  initlog     Initialise a new log.\n"
+            << "  set_config  Set/Change a cluster's config.\n";
 }
 
 
+unique_ptr<LogSigner> BuildLogSigner() {
+  CHECK(!FLAGS_key.empty());
+  util::StatusOr<EVP_PKEY*> pkey(ReadPrivateKey(FLAGS_key));
+  CHECK_EQ(pkey.status(), util::Status::OK);
+  return unique_ptr<LogSigner>(new LogSigner(pkey.ValueOrDie()));
```

Comment:
I really wish there was a ```make_unique<LogSigner>(pkey.ValueOrDie())```, this is some bad case of stuttering... The "solution" is Go, I suppose? :wink: